lib/jira/core.rb

```diff
@@ -112,11 +93,11 @@ def discard_memoized
         #
         # @param path [String] path of file to read
         #
-        # @return [String] contents of the file at the input path
+        # @return [Object] IniFile object of the file at the input path
         #
         def read(path)
           self.validate_path!(path)
-          File.read(path).strip
+          IniFile.load(path, { :comment => '#', :encoding => 'UTF-8' })
```

Comment:
I'm fine with this getting merged in for now, but as mentioned above, likely going to get rid of it completely